doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,70 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization. The HoloViz Organization is an open-source and collaborative effort with a mission to develop solutions for a wide range of visualization problems in Python.
+
+## 1. Terminology
+- "HoloViz Organization" or "Organization": The entire HoloViz ecosystem of Contributors and Projects
```

Comment:
Hmmm.

```suggestion
- "HoloViz": The entire HoloViz ecosystem of Users, Contributors,  Projects, and Libraries.
- "HoloViz Organization" or "Organization": The HoloViz Steering Committee, including representatives from each HoloViz project```

To me Organization implies a leadership or steering role. Assuming the leadership of HoloViz is distinct from the leadership of each project, having some grouping that consists of the leadership of all individual HoloViz projects plus the leadership of each project seems like it's useful to me. But I haven't thought deeply about how such a grouping would affect the rest of this PR.

Reply:
Somewhat related is whether each Project Owner (or elected Maintainer if there is no Owner) should serve on the HoloViz Committee. I'm not sure we need to codify this at the moment and I would rather retain flexibility for now, but something to think about is how Project Leadership and Steering Committee membership interact.

Reply:
How do we currently use "HoloViz Organization" in these docs? If that group is making any decisions, I don't think it can include Contributors, because making a contribution shouldn't be enough to put you in power to make decisions about the project. Any group that makes decisions about the project should have limited membership or else the group can be hijacked for purposes at odds with the project.